docs/terminology.rst

```diff
@@ -8,14 +8,11 @@ Terminology
     :header: "Name", "Description"
     :widths: 30, 60
 
-    **SpecFile**,"A ``SpecFile`` is a YAML file that buildtest interprets when generating the test. The SpecFile may
-    contain one or more specs."
-    **Spec**,"A Spec is the basic building block for writing a test specification that is compatible with one of the
-    Spec Schema."
-    **Spec Schema**,"A Spec Schema is a JSON file defining the schema with valid key/value pairs that are acceptable
-    for the schema."
-    **Schema Library**,"A Schema Library is a collection of one or more ``Spec Schema`` provided by buildtest."
-    **Test Script**,"A Test Script is a generated shell script by buildtest as a result of processing one of the SpecFile."
+    **BuildSpec**,"``BuildSpec`` is a YAML file that buildtest interprets when generating the test. A BuildSpec may
```

Comment:
Do you want that capitalization? You know, it's generally `Dockerfile` and not `DockerFile`. I'm thinking we want either `buildspec` or `Buildspec` but not `BuildSpec`

Reply:
Okay let's do ``Buildspec`` in that case it can be used at start of sentence when it get's confusing if first letter is to be caps.
Ill fix it

Reply:
Okay i made this change now